src/service/standard-actions-impl.js

```diff
@@ -113,7 +113,7 @@ export class StandardActions {
       case 'navigateTo':
         // Some components have additional constraints on allowing navigation.
         let permission = Promise.resolve();
-        if (startsWith(node.tagName, 'AMP-')) {
+        if (node.tagName && startsWith(node.tagName, 'AMP-')) {
```

Comment:
@choumx Another thing I noticed is that `node` here (target of invocation) is always "document" when an action is a global action called via `AMP.` and per https://github.com/ampproject/amphtml/blob/master/src/service/action-impl.js#L478 that seems to be the desired behaviour. So I am wondering if this line is even necessary. ?

Reply:
It's should be checking `invocation.caller.tagName`. #15693 fixes it.